stub_uploader/build_wheel.py

```diff
@@ -102,14 +97,9 @@
 DESCRIPTION_INTRO_TEMPLATE = """
 ## Typing stubs for {distribution}
 
-This is a [PEP 561](https://peps.python.org/pep-0561/)
-type stub package for the {formatted_distribution} package.
-It can be used by type-checking tools like
-[mypy](https://github.com/python/mypy/),
-[pyright](https://github.com/microsoft/pyright),
-[pytype](https://github.com/google/pytype/),
-[Pyre](https://pyre-check.org/),
-PyCharm, etc. to check code that uses `{distribution}`. This version of
+This is a [PEP 561](https://peps.python.org/pep-0561/) type stub package for
```

Comment:
What do you think of linking to the spec (https://typing.python.org/en/latest/spec/distributing.html#type-information-in-libraries) instead of PEP 561? PEPs are not ongoing documentation after all.

Reply:
There's also a link to "partial" that needs to be replaced. But let me do that in a followup PR.